channels/us.m3u

```diff
@@ -711,14 +711,14 @@ https://otteravision-tg.ottera.tv/live/master.m3u8?channel=tg_tg_us
 https://cdn20.liveonlineservices.com/hls/eye95tv.m3u8
 #EXTINF:-1 tvg-id="FailArmy.us" tvg-name="Fail Army" tvg-country="US" tvg-language="English" tvg-logo="https://assets.tvplayer.com/tvos/images/channels/585.png" group-title="General",Fail Army (720p)
 https://dai.google.com/linear/hls/event/7tuuoX1wSsCTaki1HqJFYw/master.m3u8
-#EXTINF:-1 tvg-id="FailArmyXUMO.us" tvg-name="Fail Army (XUMO)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/RIXSg4L.png" group-title="",Fail Army (XUMO)
-https://dai2.xumo.com/xumocdn/p=redbox&deviceid=&is_lat=&subp=RedboxdesktopWebLinux/amagi_hls_data_xumo1212A-redboxfailarmy/CDN/playlist.m3u8
+#EXTINF:-1 tvg-id="FailArmyXUMO.us" tvg-name="Fail Army (XUMO)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/RIXSg4L.png" group-title="Comedy",Fail Army (XUMO)
+https://live-content.xumo.com/56/content/XM036MYDW1ETSM/22452084/4.m3u8
```

Comment:
Isn't broadcast, returns a fixed chunk that ends

Reply:
This has AD Beacons. No need for a duplicate as XUMO and STIRR is the same.